pkg/daemon/update.go

```diff
@@ -1101,6 +1101,7 @@ func getSupportedExtensions() map[string][]string {
 	return map[string][]string{
 		"usbguard":     {"usbguard"},
 		"kernel-devel": {"kernel-devel", "kernel-headers"},
+		"qemu-kiwi": {"ipxe-roms-qemu", "libpmem", "pixman", "qemu-kiwi", "qemu-kvm-common", "seabios-bin", "seavgabios-bin", "sgabios-bin"},
```

Comment:
I think this line isn't formatted properly and is getting caught by the linter

Reply:
> As mentioned in the RHCOS PR, [openshift/enhancements#366](https://github.com/openshift/enhancements/pull/366) is still open, so preferably that is merged before the MCO acks the extension.

https://github.com/openshift/enhancements/pull/366 is gone.

I've opened https://github.com/openshift/enhancements/pull/677 in order to proceed with the work as Jens is out. Regardless, 677 still needs work.